wwe-service-client-api/api-overview/wwe-service-client-api.js

```diff
@@ -122,7 +122,7 @@ if (!window.genesys.wwe.service) {
         delete subscriptionsByCallback[callback];

```

Comment:
@cosnier ,
Is it normal that the path is wwe-service-client-api/api-overview/wwe-service-client-api.js?
Best regards,
Stéphane.